src/Essentials/src/SecureStorage/SecureStorage.ios.tvos.watchos.macos.cs

```diff
@@ -9,8 +9,24 @@ namespace Microsoft.Maui.Essentials
 {
 	public static partial class SecureStorage
 	{
-		public static SecAccessible DefaultAccessible { get; set; } =
-			SecAccessible.AfterFirstUnlock;
+		public static SecAccessible DefaultAccessible
+		{
+			get { return Implementations.SecureStorageImplementation.DefaultAccessible; }
+			set { Implementations.SecureStorageImplementation.DefaultAccessible = value; }
+		}
```

Comment:
And this too maybe. Feels a bit weird, but better to have this all on `(Current as ISecAccessibleSecureStorage ).DefaultAccessible`